src/javaharness/java/arcs/web/impl/WebHarnessController.java

```diff
@@ -23,12 +23,23 @@
   private final ArcsEnvironment environment;
   private final DeviceClient deviceClient;
   private final PortableJsonParser jsonParser;
+  private final DummyClipboard dummyClipboard;
+
+  public static interface DummyClipboard {
```

Comment:
done. the only caveat, I had to put it under 
java/arcs/web/impl/ rather than java/arcs/web/demo (unlike the WebClipboardService), because i need to do the ugly type cast in the WebHarnessController. Otoh i think this class is just a temporary hack and we won't need it eventually, so probably not a big deal.